.devops/templates/build-test-lint.yml

```diff
@@ -44,7 +44,11 @@ steps:
       DANGER_GITHUB_API_TOKEN: $(DANGER_GITHUB_API_TOKEN)
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31531&commitId=4b948133648a965cdf9a7b6b234ae7a9b29d280c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Popup</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popup.Popup Example Position.chromium.png|846| Changed|
|Popup.Popup Example Rtl.RTL- Shows popup.chromium.png|513| Changed|
|Popup.Popup Example Pointer Margin.chromium.png|1155| Changed|

</details>


  <details><summary><b>Tooltip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.Tooltip Example Pointer Margin.chromium.png|869| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>